<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageApp">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>365</width>
        <height>176</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Application settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxCanClose">
             <property name="text">
              <string>To tray if closed</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDatabase">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>365</width>
        <height>176</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Database</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBoxDatabases">
         <item>
          <property name="text">
           <string>SQLite</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MySQL</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxMySQL">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>MySQL configuration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditMySQLUsername"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelMySQLUsername">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelMySQLHostname">
            <property name="text">
             <string>Hostname</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditMySQLHostname"/>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEditMySQLPassword">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEditMySQLDatabase"/>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="labelMySQLPassword">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelMySQLDatabase">
            <property name="text">
             <string>Database:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageAlarmWarming">
      <attribute name="label">
       <string>Alarm warming</string>
      </attribute>
      <widget class="QPushButton" name="pushButtonOpenFile">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Open file</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxSounds">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxDatabases</tabstop>
  <tabstop>lineEditMySQLHostname</tabstop>
  <tabstop>lineEditMySQLDatabase</tabstop>
  <tabstop>lineEditMySQLUsername</tabstop>
  <tabstop>lineEditMySQLPassword</tabstop>
  <tabstop>checkBoxCanClose</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
